pkg/template/apis/template/constants.go

```diff
@@ -19,10 +19,6 @@ const (
 	// SupportURLAnnotation is the url where support for a template can be found
 	SupportURLAnnotation = "template.openshift.io/support-url"
 
-	// TemplateInstanceLabel is used to label every object created by the
-	// TemplateInstance API.
-	TemplateInstanceLabel = "template.openshift.io/template-instance"
```

Comment:
@spadgett fyi.  don't know if you were using this, but we're taking it away in preference to using ownerRefs to track the templateinstance associated with an object.

Reply:
Thanks for the heads up. We're not using it.